lib/internal/test_runner/utils.js

```diff
@@ -290,7 +290,7 @@ function parseCommandLine() {
     if (!coverageExcludeGlobs || coverageExcludeGlobs.length === 0) {
       // TODO(pmarchini): this default should follow something similar to c8 defaults
```

Comment:
Are we still planning todo c8 defaults like the comment mentions?

Reply:
@pmarchini